pages/linux/setsid.md

```diff
@@ -0,0 +1,25 @@
+# setsid
+
+> Run a program in a new session if the calling process is not a process group leader.
+> The created session is by default not controlled by the current terminal.
+> More information: <https://manned.org/setsid>.
+
+- Run a program in a new session:
+
+`setsid {{program}}`
+
+- Run a program in a new session discarding the resulting output and error:
+
+`setsid {{program}} > /dev/null 2>&1`
```

Comment:
```suggestion
`setsid {{program}} >/dev/null 2>&1`
```
I prefer this, I've never seen `&>` and don't know what it does neither how portable it is.

Reply:
I believe the space between the "`>`" and the following text is good practice, as seen on this [page](https://www.gnu.org/software/bash/manual/html_node/Redirections.html) about redirection in `bash`. 

As for the `&>`, it's a convenient way to redirect stdout and stderr, present in `bash` ([see previous link](https://www.gnu.org/software/bash/manual/html_node/Redirections.html)) and [`zsh`](https://zsh.sourceforge.io/Doc/Release/Redirection.html), but not in some other shells.